perllib/mySociety/EmailUtil.pm

```diff
@@ -84,6 +84,30 @@ sub is_valid_email ($) {
 }
 
 
+=item is_valid_email_list ADDRESSES
+
+Takes a string containing one or more comma-separated email addresses and calls
+is_valid_email on each, returning success iff they're all valid.
+
+=cut
+
+sub is_valid_email_list ($) {
+    my $addresses = shift;
+
+    my @emails;
+    if ( $addresses =~ /,/ ) {
+        @emails = split(/,/, $addresses);
+    } else {
+        @emails = ( $addresses );
+    }
+    for my $email ( @emails ) {
+        unless ( is_valid_email( $email ) ) {
+            return 0;
+        }
+    }
+    return 1;
```

Comment:
Sigh, perl on wheezy is so old, 5.20 or later we could do `return List::Util::all { is_valid_email($_) } @emails;`
You could have a one line `all` function I guess, but this is of course fine :)